### PR TITLE
Migrating Project Academic Knowledge documentation to our repo

### PR DESCRIPTION
**Pull request by DarrinEide**
This PR is for the migration of https://docs.microsoft.com/en-us/azure/cognitive-services/academic-knowledge/home to a new hub in our own repository.

You can preview the changes here:
https://review.docs.microsoft.com/en-us/academic-services/project-academic-knowledge/introduction?branch=DarrinEide%2Fproject-academic-knowledge-migration

### CODE REVIEW

File: academic-services/breadcrumb/toc.yml [hunk 1]
```diff
@@ -7,4 +7,7 @@
     topicHref: /academic-services/graph/
   - name: Microsoft Academic Knowledge Exploration Service
     tocHref: /academic-services/knowledge-exploration-service/
-    topicHref: /academic-services/knowledge-exploration-service/
\ No newline at end of file
+    topicHref: /academic-services/knowledge-exploration-service/
+  - name: Project Academic Knowledge
+    tocHref: /academic-services/project-academic-knowledge/
+    topicHref: /academic-services/project-academic-knowledge/
```
Comment (alch-msft) [inline academic-services/breadcrumb/toc.yml]: Does not resolve

File: academic-services/project-academic-knowledge/reference-affiliation-entity-attributes.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Affiliation entity attributes
```
Comment (alch-msft) [inline academic-services/project-academic-knowledge/reference-affiliation-entity-attributes.md]: Should we refer to **https://review.docs.microsoft.com/en-us/academic-services/knowledge-exploration-service/reference-makes-api-entity-schema?branch=DarrinEide%2Fproject-academic-knowledge-migration** for entity schema so we don't have to maintain this in two places?

File: academic-services/project-academic-knowledge/toc.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    items:
+      - name: CalcHistogram
+        href: reference-calchistogram-method.md
+      - name: Evaluate
+        href: reference-evaluate-method.md
+      - name: Interpret
+        href: reference-interpret-method.md
+      - name: Similarity
+        href: reference-similarity-method.md
+  - name: Microsoft Research APIs
```
Comment (alch-msft) [inline academic-services/project-academic-knowledge/toc.yml]: Should this be under **Resources**? 
Should we call it "Microsoft Research API Portal" so readers don't confuse it with "REST API"?

File: academic-services/project-academic-knowledge/introduction.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  4. **similarity** - Calculates the cosine similarity between two strings
+  
+Used together, these API methods allow you to create a rich semantic search experience. Given a user query string, the **interpret** method provides you with an annotated version of the query and a structured query expression, while optionally completing the user’s query based on the semantics of the underlying academic data. For example, if a user types the string *latent s*, the **interpret** method can provide a set of ranked interpretations, suggesting that the user might be searching for the field of study *latent semantic analysis*, the paper *latent structure analysis*, or other entity expressions starting with *latent s*. This information can be used to quickly guide the user to the desired search results.
+
+The **evaluate** method can be used to retrieve a set of matching paper entities from the academic knowledge base, and the **calchistogram** method can be used to calculate the distribution of attribute values for a set of paper entities which can be used to further filter the search results.
+
+## Getting Started
+
```
Comment (alch-msft) [inline academic-services/project-academic-knowledge/introduction.md]: We should add a small paragraph about signing up via Microsoft Research API portal

File: academic-services/project-academic-knowledge/toc.yml (hunk 3 above)
Comment (DarrinEide) [reply academic-services/project-academic-knowledge/toc.yml]: I was on the fence, but yea I can see that it should prob be there

---
In reply to: [383564519](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/119#discussion_r383564519) [](ancestors = 383564519)

File: academic-services/project-academic-knowledge/introduction.md (hunk 4 above)
Comment (DarrinEide) [reply academic-services/project-academic-knowledge/introduction.md]: Let's park that idea for now. I want to first see how receptive the team is for us controlling our own documentation first.

---
In reply to: [383564919](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/119#discussion_r383564919) [](ancestors = 383564919)

File: academic-services/project-academic-knowledge/reference-affiliation-entity-attributes.md (hunk 2 above)
Comment (DarrinEide) [reply academic-services/project-academic-knowledge/reference-affiliation-entity-attributes.md]: That's my eventual goal, but that can wait till post-migration

---
In reply to: [383562916](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/119#discussion_r383562916) [](ancestors = 383562916)

**Pull request merged**